samples/Common.c

```diff
@@ -563,16 +563,29 @@ STATUS handleRemoteCandidate(PSampleStreamingSession pSampleStreamingSession, PS
     return retStatus;
 }
 
+STATUS traverseDirectoryPEMFileScan(UINT64 customData, DIR_ENTRY_TYPES entryType, PCHAR fullPath, PCHAR fileName) {
+    PCHAR certName = (PCHAR) customData;
+    INT32 fileNameLen = STRLEN(fileName);
+
+    if(fileNameLen > ARRAY_SIZE(CA_CERT_PEM_FILE_EXTENSION) + 1 &&
+       (STRCMPI(CA_CERT_PEM_FILE_EXTENSION, &fileName[fileNameLen - ARRAY_SIZE(CA_CERT_PEM_FILE_EXTENSION) + 1]) == 0))
+    {
+        certName[0] = FPATHSEPARATOR;
+        certName++;
+        STRCPY(certName, fileName);
```

Comment:
I think there is a way to terminate the iterative - was it returning error or something

Reply:
I double checked and just a `CHK_STATUS` so I don't believe so? happy to make a backlog item out of it though!